test/testdrive/session.td

```diff
@@ -68,6 +68,7 @@ max_tables                               200                     "The maximum nu
 network_policy                           default                 "Sets the fallback network policy applied to all users without an explicit policy."
 optimizer_e2e_latency_warning_threshold  "500 ms"                "Sets the duration that a query can take to compile; queries that take longer will trigger a warning. If this value is specified without units, it is taken as milliseconds. A value of zero disables the timeout (Materialize)."
 mz_version                               <VARIES>                "Shows the Materialize server version (Materialize)."
+password_hash_iterations                 600000                  "Iterations to use when hashing passwords. Higher iterations are more secure, but take longer to validated. Please consider the security risks before reducing this below the default value."
```

Comment:
Kind of feel like we should just keep parity with postgresql and call this `scram_iterations`